src/content/developers/docs/oracles/index.md

```diff
@@ -4,61 +4,61 @@ description: Oracles provide Ethereum smart contracts with access to real-world
 lang: en
 ---
 
-Oracles are data feeds that bring data from off the blockchain (off-chain) data sources and puts it on the blockchain (on-chain) for smart contracts to use. This is necessary because smart contracts running on Ethereum cannot access information stored outside the blockchain network.
+Oracles are data feeds that make data from off-chain sources available to the blockchain for smart contracts to use. This is necessary because Ethereum-based smart contracts cannot, by default, access information stored outside the blockchain network.
 
-Giving smart contracts the ability to execute using off-chain data inputs extends decentralized applications' value. For instance, decentralized prediction markets rely on oracles to provide information about outcomes with which they can validate user predictions. Suppose Alice bets 20 ETH on who will become the next U.S. President. In that case, the prediction-market dapp needs an oracle to confirm election results and determine if Alice is eligible for a payout.
+Giving smart contracts the ability to execute using off-chain data inputs extends the utility and value of decentralized applications. For instance, on-chain prediction markets rely on oracles to provide information about outcomes that they use to validate user predictions. Suppose Alice bets 20 ETH on who will become the next U.S. President. In that case, the prediction-market dapp needs an oracle to confirm election results and determine if Alice is eligible for a payout.
```

Comment:
Is the word 'inputs' necessary here? 🤔 

```suggestion
Giving smart contracts the ability to execute using off-chain data extends the utility and value of decentralized applications. For instance, on-chain prediction markets rely on oracles to provide information about outcomes they use to validate user predictions. Suppose Alice bets 20 ETH on who will become the next U.S. President. In that case, the prediction-market dapp needs an oracle to confirm election results and determine if Alice is eligible for a payout.
```

Reply:
nope, and in general any changes you prefer I'm good with -- editing is a lot easier than writing IMO (my favorite TV writer makes this point beautifully here https://www.newyorker.com/culture/the-new-yorker-interview/john-swartzwelder-sage-of-the-simpsons) so the real credit goes to the person who drafted the material, from here all small-ish suggestions like this are good by me

Reply:
> editing is a lot easier than writing IMO

Agree 100%! I find it incredibly hard to edit my own work (ego? 😬).

Will have a look over again soon and get this in. Overall at a glance, lots of nice changes ❤️